storey/targets.py

```diff
@@ -547,6 +549,7 @@ def __init__(
         infer_columns_from_data: Optional[bool] = None,
         max_events: Optional[int] = None,
         flush_after_seconds: Union[int, float, None] = None,
+        single_file: bool = False,
```

Comment:
```suggestion
        single_file: Optional[bool] = None,
```
This needs to be done for mlrun serialization, which is the reason the rest of the parameters use this pattern.